src/js/bs3/ui.js

```diff
@@ -165,6 +168,7 @@ define([
     },
 
     createLayout: function ($note, options) {
+      self.options = options;
```

Comment:
I can't find a variable named `self` in this scope. I thought this code seems to be dangerous.